app/gui/view/graph-editor/src/component/node/action_bar.rs

```diff
@@ -31,9 +33,11 @@ const BUTTON_OFFSET: f32 = 0.5;
 /// Grow the hover area in x direction by this amount. Used to close the gap between action
 /// icons and node.
 const HOVER_EXTENSION_X: f32 = 15.0;
+const VISIBILITY_TOOLTIP_LABEL: &str = "Show preview";
+const DISABLE_OUTPUT_CONTEXT_TOOLTIP_LABEL: &str = "Disable output execution";
+const ENABLE_OUTPUT_CONTEXT_TOOLTIP_LABEL: &str = "Enable output execution";
```

Comment:
```suggestion
const DISABLE_OUTPUT_CONTEXT_TOOLTIP_LABEL: &str = "Disable output context";
const ENABLE_OUTPUT_CONTEXT_TOOLTIP_LABEL: &str = "Enable output context";
```

Reply:
I'm happy to change it, but I thought that "output context" might not be a meaningful term to our average user. It's certainly not clear to me. Perhaps @wdanilo has some product insights here?

Reply:
I think the term `execution` is not meaningful here either, as we do not control the execution. E.g. the "disabled" node is still execute, but with different permissions.

Reply:
It does change part of it's execution, doesn't it? I thought the underlying code would have things like "if we have this permission, do this, otherwise do that"?

As an alternative, perhaps we could say "disable/enable output side effects"?

Reply:
Side effect is probably the most accurate term. @wdanilo what do you think?

Reply:
I think we could use something like "Allow writing to files and databases" instead - this should be understandable without introducing new terms to users. What do you think? :)

Reply:
Sounds good to me. Thanks, @wdanilo!